src/components/ChatEntry.js

```diff
@@ -1,22 +1,37 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
+  const handleToggleLike = () => {
+    const updatedEntry = {
+      id  : props.id,
+      sender : props.sender,
+      body: props.body,
+      timeStamp : props.timeStamp,
+      liked : !props.liked
+    }
+    props.toggleLike(updatedEntry)
+  }
```

Comment:
When a user clicks on the heart we only need to update the liked property. We should always prefer to minimize the scope of our changes to data and only change data that absolutely needs to be changed.

Here, handleToggleLike object is resetting the values of id, sender, body, timestamp, and liked but none of the values on lines 9-12 change when a user clicks the heart button.

How would you refactor this method and updateToggleLike in App.js so you don't need to change the whole message?